client/blocks/image-editor/index.jsx

```diff
@@ -95,6 +104,15 @@ const ImageEditor = React.createClass( {
 
 			this.setDefaultAspectRatio();
 		}
+
+		if ( newProps.originalAspectRatio && ! isEqual( newProps.originalAspectRatio, originalAspectRatio ) ) {
+			if ( newProps.originalAspectRatio.width > MinimumImageDimensions.WIDTH &&
+				newProps.originalAspectRatio.height > MinimumImageDimensions.HEIGHT ) {
+				this.setState( {
+					isGreaterThanMinimumDimensions: true
```

Comment:
Good suggestion @marcinbot. Let's remember to add the new selector to the global folder `client/state/selectors`. We probably don't need to memo in this case, but if you're curious here's our [lib](https://github.com/Automattic/wp-calypso/tree/master/client/lib/create-selector) that helps us with this.